packages/cashier/src/modules/deposit-fiat/deposit-fiat.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useFetch } from '@deriv/api';
+import { Loading } from '@deriv/components';
+import { PageContainer } from '../../components/page-container';
+import CashierOnboardingSideNotePaymentMethods from '../cashier-onboarding/components/cashier-onboarding-side-notes/cashier-onboarding-side-note-payment-methods';
+import './deposit-fiat.scss';
+
+const DepositFiatModule = () => {
+    const [show_loader, setShowLoader] = React.useState(true);
+    const { data, isSuccess } = useFetch('cashier', {
```

Comment:
@aum-deriv Let's make a `useDepositFiatAddress` hook for it, Similar to `useDepositCryptoAddress` we have

Reply:
@farzin-deriv Do I have to use `useRequest` hook from api package for this? OR useFetch is fine?

Reply:
@aum-deriv `useFetch` will cache it, I'm not sure but I think in your case it can be cashed 🤔

Would you check this with someone from BE and ask if it's okay to cache the response or not 🙏🏻

Reply:
@farzin-deriv I think the logic for `useDepositCryptoAddress` can be used for both fiat and crypto. Can we generalize it into one hook?

Reply:
@aum-deriv Let's create a separate one, It doesn't make sense both of them are the same endpoint actually, Maybe at some point, we can ask the backend for improvement to separate them so better we keep them separated already :D